tests/functional_tests/ffront_tests/test_lowering.py

```diff
@@ -251,6 +253,41 @@ def plus(a: Field[..., "float64"], b: Field[..., "float64"]):
     assert lowered.expr == reference
 
 
+def test_add_scalar_literal_to_field():
+    def scalar_plus_field(a: Field[..., "float64"]) -> Field[..., "float64"]:
+        return 2.0 + a
+
+    parsed = FieldOperatorParser.apply_to_function(scalar_plus_field)
+    lowered = FieldOperatorLowering.apply(parsed)
+
+    reference = im.deref_(
+        im.lift_(im.lambda__("a")(im.plus_(im.number_("float64", "2.0"), im.deref_("a"))))("a")
+    )
+
+    assert lowered.expr == reference
+
+
+def test_add_scalar_literals():
+    def scalar_plus_scalar(a: Field[..., "int32"]) -> Field[..., "int32"]:
+        tmp = int32(1) + int32(1)
```

Comment:
I am not sure what this would test that isn't tested between `test_add_scalar_literal_to_field` and `test_add_scalar_literals`. Do you mean to test that the *result* of field + scalar can be added to a scalar?

Reply:
No, my reasoning was that I didn't see a test where there is an assignment with scalar + field.